src/resolvers-cassandra/Messages/queries.js

```diff
@@ -12,26 +12,33 @@ const trackEvent = require('../../clients/appinsights/AppInsightsClient').trackE
  */
 
 function eventToFeature(row) {
+  const FeatureType = 'MultiPoint';
+
   return {
-    type: row.pipelinekey,
-    coordinates: [],
+    type: FeatureType,
+    coordinates: row.computedfeatures.places.map(place=>[place.centroidlon, place.centroidlat]),
```

Comment:
I'd look into this more since I think Clemens may have an implementation that can be reused.